hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -1035,6 +1035,40 @@ def stop_charge(self, token: Token, vehicle: Vehicle) -> str:
         token.device_id = self._get_device_id(self._get_stamp())
         return response["msgId"]
 
+    def start_hazard_lights(self, token: Token, vehicle: Vehicle) -> str:
+        url = self.SPA_API_URL_V2 + "vehicles/" + vehicle.id + "/ccs2/control/light"
+
+        payload = {"command": "on"}
+        _LOGGER.debug(f"{DOMAIN} - Start Hazard Lights Request: {payload}")
+        response = requests.post(
+            url,
+            json=payload,
+            headers=self._get_authenticated_headers(
+                token, vehicle.ccu_ccs2_protocol_support
+            ),
+        ).json()
+        _LOGGER.debug(f"{DOMAIN} - Start Hazard Lights Response: {response}")
+        _check_response_for_errors(response)
+        token.device_id = self._get_device_id(self._get_stamp())
+        return response["msgId"]
+
+    def start_hazard_lights_and_horn(self, token: Token, vehicle: Vehicle) -> str:
+        url = self.SPA_API_URL_V2 + "vehicles/" + vehicle.id + "/ccs2/control/hornlight"
+
+        payload = {"command": "on"}
+        _LOGGER.debug(f"{DOMAIN} - Start Hazard Lights and Horn Request: {payload}")
+        response = requests.post(
+            url,
+            json=payload,
+            headers=self._get_authenticated_headers(
+                token, vehicle.ccu_ccs2_protocol_support
+            ),
```

Comment:
I am getting `Unknown API response format: {'error': 'Access to this API has been disallowed'}`

but it works fine after change the 'headers' from self._get_authenticated_headers to 

```
            headers=self._get_authenticated_headers(
                token, vehicle.ccu_ccs2_protocol_support
            ),
```

to 

```
headers=self._get_control_headers(
                token, vehicle
            ),
```

It should then replicate the PUT request for the pin before the hazard light command.

Reply:
Updated - can you check now for me @asychow? I'm still rate limited.

Reply:
Works on my end :)